app/src/pages/[resourceType]/search-result.tsx

```diff
@@ -75,16 +76,7 @@ export const getServerSideProps: GetServerSideProps<{
     const resources = bundle.entry;
     const resourceInfoArray = resources.reduce((acc: ResourceInfo[], entry) => {
       if (entry.resource && entry.resource.id) {
-        const identifier = entry.resource.identifier?.[0];
-        const resourceInfo: ResourceInfo = {
-          resourceType: checkedResourceType,
-          id: entry.resource.id,
-          identifier: identifier?.system && identifier?.value ? `${identifier.system}|${identifier.value}` : null,
-          name: entry.resource.name ?? null,
-          url: entry.resource.url ?? null,
-          version: entry.resource.version ?? null,
-          status: entry.resource.status ?? null
-        };
+        const resourceInfo = extractResourceInfo(entry.resource);
```

Comment:
Currently, should the search functionality also take into consideration draft artifacts?

Reply:
Right now, the search functionality should only take artifacts from the service into account. In the future, we might want to add a separate search for drafts, or even combine search results for drafts and active artifacts, but I think that's outside of the scope of this task because it will require some more discussion to figure out exactly what we want to do. We have a task in the backlog for refactoring some of the search results to de-duplicate some of the code, so we could bring this up when we start exploring that task.